client/src/SharedComponents/ProjectView/ProjectView.js

```diff
@@ -1,10 +1,16 @@
 import React from 'react';
 import './ProjectView.css';
 
-const ProjectView = ({ projectImg }) => {
+const ProjectView = ({ projectImg, projectDetailsStyle }) => {
   return (
-    <img className='project-view' src={projectImg} alt="" />
-  )
-}
+    <div>
+      {projectDetailsStyle ? (
+        <img className='project-imgview' src={projectImg} alt='' />
+      ) : (
```

Comment:
@kholoudfann Since most of the img tags is the same, you could reduce some duplication by putting the className in a const:
```
const ProjectView = ({ projectImg, projectDetailsStyle }) => {
  const className = projectDetailsStyle ? 'project-imgview' : 'project-view';
  return <img className={className} src={projectImg} alt='' />;
})
```